### DISCUSSION TITLE
Suggestion to alter current Logo

### DISCUSSION THREAD

**Discussion by bubdrum** [Glimesh/company#13] [Ideas]
We have received some suggestions to possible alter the current Glimesh logo to include a white stroke (outline) around it. 

Below is an example of what it would look like.  Please provide your thoughts on this proposed change.
**Edit** Received further info that the colors of the logo are changed as well as the complimenting background color**

![outlinelogo](https://user-images.githubusercontent.com/76960703/104759913-98e21e80-572e-11eb-9039-68ebab19f08e.png)

**Comment by Heiwa1580** [comment]
I personally am not a huge fan on the white outline. As it really only affects the logo in dark mode. 

It would be interesting to see a mockup of the whole site in the new colors as that could change the whole visual appearance of our color scheme on the site which could affect other areas. But I do think they help the logo stand out a little more. It doesn't get lost in the background as much.

With the shading going the other way it does make the triangle to appear to point another direction. Not that big of an issue but it could affect the way our logo looks with the text to the side.

![image](https://user-images.githubusercontent.com/68549151/104774801-4c560d80-5745-11eb-849a-cf20ae02fff5.png)

I personally don't think we need to necessarily change the whole logo colors and scheme. But perhaps just change the background color behind the logo alone. I think we could make some big changes just by doing that. Perhaps a light heather gray than a dark navy blue

**Reply by Heiwa1580** [reply]
![image](https://user-images.githubusercontent.com/68549151/104775493-67754d00-5746-11eb-8540-03a8ea988221.png)

This is what I was thinking but perhaps a darker shade would be better haha

**Reply by justsimy** [reply]
Maybe these # codes? # 193573 # 20418C # 143580 # 082D81 ( hex codes ) as per your suggestion Heiwa on background color behind logo
